src/Test/Runner/Log.elm

```diff
@@ -0,0 +1,76 @@
+module Test.Runner.Log exposing (run, runWithOptions)
+
+{-| # Log Runner
+
+Runs a test and outputs its results using `Debug.log`, then calls `Debug.crash`
+if there are any failures.
+
+This is not the prettiest runner, but it is simple and cross-platform. For
+example, you can use it as a crude Node runner like so:
+
+    $ elm-make LogRunnerExample.elm --output=elm.js
+    $ node elm.js
+
+This will log the test results to the console, then exit with exit code 0
+if the tests all passed, and 1 if any failed.
+
+@docs run, runWithOptions
+-}
+
+import Random
+import Test exposing (Test)
+import Test.Runner.String exposing (Summary)
+import String
+
+
+{-| Run the test using the default `Test.Runner.String` options.
+-}
+run : Test -> a -> a
```

Comment:
I'd like for there to be a default, but I think it's important that they live in different packages so they can be versioned differently.

For example, if `elm-test` depends on `html`, and my app does too, it's possible to find myself in a situation where I can no longer use the latest `elm-test` because I have not yet upgraded my application to use the latest `html`...even though I am not using the HTML test runner.

(We ran into this back in like the 0.15-0.16 days. It was a pain.)

My intention was to provide (and maintain myself) the default implementations and link to them from the README. Maybe I could use them for the examples too, come to think of it...at which point there's no need for the `String` runner.